src/metabase/driver/postgres.clj

```diff
@@ -85,19 +82,6 @@
     ;; only supported for Postgres for right now. Not supported for child drivers like Redshift or whatever.
     (= driver :postgres)))
 
-(defn- ->timestamp [honeysql-form]
-  (hx/cast-unless-type-in "timestamp" #{"timestamp" "timestamptz" "date"} honeysql-form))
-
-(defmethod sql.qp/add-interval-honeysql-form :postgres
```

Comment:
Moved this so it's in the `metabase.driver.sql` impls section along with ALL of the other `metabase.driver.sql` multimethods impls.